src/client/dfuse/dfuse_cont.c

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright 2016-2021 Intel Corporation.
+ * (C) Copyright 2016-2022 Intel Corporation.
```

Comment:
I don't know how these slipped past, but it shouldn't be modified in this PR.

I've locally modified the update-copyright hook to not run on merge commits so maybe that's why this is happening here?

Reply:
I think at one point Jeff fixed the hook, which might have been after this PR was originally created